doc/developer-guides/hld/hv-console.rst

```diff
@@ -70,10 +70,24 @@ Specifically:
    the hypervisor shell. Inputs to the physical UART will be
    redirected to the vUART starting from the next timer event.
 
--  The vUART is deactivated after a :kbd:`Ctrl` + :kbd:`Space` hotkey is received
-   from the physical UART. Inputs to the physical UART will be
-   handled by the hypervisor shell starting from the next timer
-   event.
+-  The vUART enters escaping mode after a BREAK character is received from
+   the physical UART. When in escaping mode, next received character will
+   be a command. After processing this command, the vUART exits the escaping
+   mode. So far, following escaping commands are supported:
+
+   - BREAK charater. If user sends break charater again in escaping mode,
+     one break charater will be sent to vUART.
+
+   - Character "e". This will deactive vUART. Inputs to the physical UART will
+     be handled by the hypervisor shell starting from the next timer event.
+
+   Other characters are not supported. The physical UART will prompt out an
+   "Unknown escaping key" message and the active vUART exits escaping mode.
+
+   Note that the BREAK character is a control character and different serial
+   terminals have different ways to send it, for example, `<ctrl-a> + F`
+   in minicom, `<ctrl-a> + <ctrl-|>` in picocom, right click -> special
```

Comment:
In picocom, Break key is `C-a + C-\`

Reply:
<c-a>+<c-h>shows:

*** Picocom commands (all prefixed by [C-a])

*** [C-x] : Exit picocom
*** [C-q] : Exit without reseting serial port
*** [C-b] : Set baudrate
*** [C-u] : Increase baudrate (baud-up)
*** [C-d] : Decrease baudrate (baud-down)
*** [C-i] : Change number of databits
*** [C-j] : Change number of stopbits
*** [C-f] : Change flow-control mode
*** [C-y] : Change parity mode
*** [C-p] : Pulse DTR
*** [C-t] : Toggle DTR
*** [C-|] : Send break
*** [C-c] : Toggle local echo
*** [C-s] : Send file
*** [C-r] : Receive file
*** [C-v] : Show port settings
*** [C-h] : Show this message

Reply:
>  C-backslash
>             Generate a break sequence on the serial line.   A  break  sequence  is  usually  generated  by  marking
>             (driving  to  logical  one)  the serial Tx line for an amount of time coresponding to several character
>             durations.

`man picocom` gives this

Reply:
`|` and `\` share the same key.  Maybe `<c-|>` and `<c-\>` are the same, like `<c-a>` and `<c-A>`.

Reply:
Let's be accurate here. `C-A` can mean `Control + Shift + a` and is thus different from `C-a`.

If picocom supports both, we can simply use `C-\`.

Reply:
ok